sagemaker_studio_image_build/cli.py

```diff
@@ -70,6 +78,15 @@ def main():
         "--repository",
         help="The ECR repository:tag for the image (default: sagemaker-studio-${domain_id}:latest)",
     )
+    build_parser.add_argument(
+        "--image",
+        help="The ECR repository:tag for the image (default: sagemaker-studio-${domain_id}:latest)",
+    )
+    build_parser.add_argument(
+        "--compute-type",
+        help="The code build compute type (default: BUILD_GENERAL1_SMALL)",
```

Comment:
nitpick: s/code build/CodeBuild
```suggestion
        help="The CodeBuild compute type (default: BUILD_GENERAL1_SMALL)",
```